packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -53,6 +57,10 @@ const GENERATED_ANSWER_DATA_KEY = 'coveo-generated-answer-data';
  * <c-quantic-generated-answer engine-id={engineId} with-toggle collapsible></c-quantic-generated-answer>
  */
 export default class QuanticGeneratedAnswer extends LightningElement {
+  constructor() {
+    super();
+    this._maxCollapsedHeight = DEFAULT_COLLAPSED_HEIGHT;
+  }
```

Comment:
Why this?
while you can simply define the property `_maxCollapsedHeight ` and give it an initial value outside the constructor.


it can simply be this line:
```
  /** @type {number} */
  _maxCollapsedHeight = DEFAULT_COLLAPSED_HEIGHT;
```